docs/gcloud-auth.rst

```diff
@@ -27,6 +27,10 @@ Overview
 
         $ export GOOGLE_APPLICATION_CREDENTIALS="/path/to/keyfile.json"
 
+    The default project ID will be the project that keyfile.json was genereated
+    from. This can be overridded by setting the ``GCLOUD_PROJECT`` environment
+    variable.
```

Comment:
Yes. I also just realized this comment is in `gcloud-auth`. The project is not related to auth, so please put it in a different file. Also make `keyfile.json` in a fixed-width font / `<code>` block, "genereated" is mis-spelled and overridded / overrided isn't a word, maybe you meant to type overridden?

Instead of referring to `keyfile.json`, just call it the JSON keyfile.